fiftyone/core/collections.py

```diff
@@ -5430,7 +5430,9 @@ def select_by(self, field, values, ordered=False):
         )
 
     @view_stage
-    def select_fields(self, field_names=None, _allow_missing=False):
+    def select_fields(
+        self, field_names=None, _allow_missing=False, meta_filter=None
```

Comment:
nit: put `meta_filter` before `_allow_missing` so that all public kwargs come before private ones. 

(by convention, we're okay with changing the order of kwargs; we assume users use `key=value` for all optional args)